test/api/unit/serializers/site.test.js

```diff
@@ -1,8 +1,9 @@
-const { expect } = require('chai');
+const { expect, assert } = require('chai');
 const validateJSONSchema = require('jsonschema').validate;
 
 const siteSchema = require('../../../../public/swagger/Site.json');
 const factory = require('../../support/factory');
+const { Domain, Site } = require('../../../../api/models');
```

Comment:
`Site` doesn't like it is used.

Reply:
True. Probably an artifact of an uncommitted early state of my work.